tools/oldest_constraints.txt

```diff
@@ -26,6 +26,9 @@ zope.event==4.0.3
 zope.interface==4.0.5
 
 # Debian Jessie Backports constraints
+# Debian Jessie has reached end of life. However:
+# When updating these dependencies, you should only update them to the oldest version of the package found
+# in a non-EOL'd version of CentOS, Debian, or Ubuntu that has Certbot packages in their OS repositories.
```

Comment:
```suggestion
# in a non-EOL'd version of CentOS, Debian, or Ubuntu that Certbot is supported on.
```

Reply:
Erica I think whatever you want here is fine, but FYI:

1) The exact wording for this entire sentence came from me.
2) While maybe overly verbose, it really is only distros that have Certbot OS packages. This file only contains Python packages and for other distribution methods such as certbot-auto, Docker, or snaps, we download our own versions from PyPI.

Reply:
I didn't realize about that latter bit; resolving this conversation.